solutions/usaco-965.mdx

```diff
@@ -88,4 +91,51 @@ int main() {
 		cout << cows[t] << '\n';
 	}
 }
-```
\ No newline at end of file
+```
+</CPPSection>
+
+<PySection>
+## Brute Force with Itertools
+
+<Warning>
+Due to the brute force nature of this solution, it is very slow and is not the optimal solution.
+</Warning>
+We can easily iterate through every possible permutation of cows in lexographic order using Itertools. Then we can check for the first permutation that meets the adjacency constraints and this will be our answer. 
+
+**Time Complexity:** $\mathcal{O}(NC*C!)$, where $C$ represents the number of cows 
+```py
+NUM_COWS = 8
+from itertools import permutations
+
```

Comment:
i'm not sure about python conventions, but shouldn't import statements usually come first?

Reply:
uhh honestly i have no idea @SansPapyrus683 @ryanchou-dev

Reply:
yeah imports come first

Reply:
k